lib/log.rb

```diff
@@ -183,6 +183,21 @@ def reset_dedup
       @dup_buffer.reset if @dup_buffer
     end
 
+    private
+    def sensor(msg)
+      secured_lines = []
+      lines = msg.split("\n")
+      sensor_kw = %W{client_id client_secret subscription_id tenant_id access_key_id secret_access_key secret authorization password oauth token .dockercfg .dockerconfigjson kubeconfig htpasswd ca service-ca tls tls service-account service_account serviceaccount sa}
```

Comment:
If there is client-id, how to cover? Suggestion: maybe using `id\b?` has fuller coverage? Tested:
```
irb(main):026:0> str="client_id.xxx: abc"
=> "client_id.xxx: abc"
irb(main):028:0> str.match(/id\b?.*:.*/)
=> #<MatchData "id.xxx: abc">

irb(main):030:0> str2="client_id_xxx.xxx: abc"
=> "client_id_xxx.xxx: abc"
irb(main):031:0> str2.match(/id\b?.*:.*/)
=> #<MatchData "id_xxx.xxx: abc">
```
Same for the remaining: just use key\b? secret\b? token\b? account\b? etc.

Reply:
Thanks! let me update and test it.

Reply:
Having shorter keywords like `id` would overprotect data in the log, as I tested, this may not help troubleshooting, for example: 
```
      metadata:
        annotations:
          openshift.io/description: ""
          openshift.io/display-name: ""
          openshift.io/requester: testuser-0
          openshift.io/sa.scc.mcs: [PROTECTED_DATA]
          openshift.io/sa.scc.supplemental-groups: [PROTECTED_DATA]
          openshift.io/sa.scc.uid-range: [PROTECTED_DATA]
        creationTimestamp: "2021-11-25T06:13:55Z"
        labels:
          kubernetes.io/metadata.name: sxgj9
        name: sxgj9
        resourceVersion: "50033"
        uid: [PROTECTED_DATA]
```

Reply:
Ah, OK, thanks for showing this. Given shorter keywords id, sa are not good, **then \b is not needed either** (\b is used to include xxxid like client-/_id but exclude idxxx like ideas)

Reply:
Sure, thank you